### PR TITLE
Dikshit/chimera load bank power supply upgrade

### PR DESCRIPTION
**Pull request by DJ90864**
### Changelist 
Updated Chimera, Added Python Class for RIGOL DC Load Bank and Power Supply.
Also added some examples.

### Testing Done
Simple testing with both Load Bank and Power Supply has been done and no issues so far.

### Resolved Tickets
- [FIRM-70](https://ubcformulaelectric.atlassian.net/browse/FIRM-70)

[FIRM-70]: https://ubcformulaelectric.atlassian.net/browse/FIRM-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: validationtools/validationtools/loadBank/_loadBank.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+class LoadBank:
+    def __init__(self):
+        '''
+        Represents a Rigol DL3021 programmable DC electronic load.
+        '''
+        resources = visa.ResourceManager()
+
+        resourceTag = ''
+        for x in resources.list_resources():
+            print(x)
```
Comment (Lucien950) [inline validationtools/validationtools/loadBank/_loadBank.py:17]: this print should be probably left to only debug/verbose mode. Make use of `logging` python library for that.

**Review by Lucien950 — APPROVED** [review]
ok mostly chill, just a few things to clean before merging. I'll trust you that the query paramters are correct.

**Comment by Lucien950** [comment]
Also in the future I think this task was large enough to warrant a ticket in Jira.

File: validationtools/validationtools/loadBank/_loadBank.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            print("No DL3021 detected.")
+            return
+
+        self.load = resources.open_resource(resourceTag)
+
+    def set_current(self, current):
+        """
+        Sets the current for the load in constant current (CC) mode.
+
+        current: float, desired current in Amps
```
[the remark this answers is not in the document]
Comment (DJ90864) [reply validationtools/validationtools/loadBank/_loadBank.py:32]: done!

File: validationtools/validationtools/loadBank/_loadBank.py (hunk 1 above)
Comment (DJ90864) [reply validationtools/validationtools/loadBank/_loadBank.py]: done!

**Pull request merged**